react-nd/src/components/Button/Button.module.scss

```diff
@@ -0,0 +1,27 @@
+.button {
+  padding: 1rem 1.5rem;
+  background-color: #8056eb;
+  color: #ffffff;
+  border-radius: 8px;
+  font-size: 1rem;
+
+  &:hover {
+    background-color: #5b24e7;
+  }
+}
+
+.rounded {
+  max-width: 4rem;
+  max-height: 4rem;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@media (max-width: 700px) {
```

Comment:
naudokite mixins'us, tam kad išvengti gausos taškų, o būtų naudojami konfiguruojami keli lūžio taškai:
https://dev.to/rembertdesigns/how-to-write-media-queries-with-sass-mixins-32e